extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/IcebergCommands.scala

```diff
@@ -49,10 +49,22 @@ object IcebergCommands {
     TableCommandSpec(cmd, Seq(tableDesc), queryDescs = Seq(queryDesc))
   }
 
+  val IcebergCallCommand = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.Call"
+    val actionTypeDesc = ActionTypeDesc(actionType = Some(INSERT))
```

Comment:
`UPDATE` is the more proper one for the action type.

Reply:
Thanks, done